torch/csrc/jit/runtime/static/generated_ops.cpp

```diff
@@ -2637,6 +2637,27 @@ REGISTER_OPERATOR_FUNCTOR(
       return nullptr;
     });
 
+REGISTER_OPERATOR_FUNCTOR(
```

Comment:
Interesting -- was there a test or build failure that required this PR to add this?

Reply:
Not needed. This should be handled by codegen.